docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -689,18 +689,24 @@ rule read from top to bottom. If no rules are specified, all access to system
 information is denied. If no rule matches, system access is denied. Each rule is
 composed of the following fields:
 
+* ``role`` (optional): regex to match against role. If matched, it
+  grants or denies the authorization based on the value of ``allow``.
 * ``user`` (optional): regex to match against user name. If matched, it
   grants or denies the authorization based on the value of ``allow``.
 * ``allow`` (required): set of access permissions granted to user. Values:
   ``read``, ``write``
 
-For example, if you want to allow only the role ``admin`` to read and write
-system information, allow ``alice`` to read system information, and deny all
-other access, you can use the following rules:
+The following configuration provides and example:
 
 .. literalinclude:: system-information-access.json
     :language: json
 
+* All users with the role ``admin`` have read and write access to system
+  information. This includes the ability to trigger
+  :doc:`/admin/graceful-shutdown`.
+* The user ``alice`` can read system information.
```

Comment:
We could give an example what read system information means for user. That this is used to read stats about nodes (GET /v1/node)

Reply:
Ok .. let me do that

Reply:
Done in separate commit as well.. actually not here but higher up in the same section and with more details. Please confirm those.